### DISCUSSION TITLE
#3242 在程序内部是怎么触发的

### DISCUSSION THREAD

**Discussion by allMagicNB** [Meloong-Git/PCL#3244] [讨论]
个人猜想……处理API的URL时要转码，然后转码太多次导致的卡住、占用CPU？
原Issue已经Locked了，我说一句，Author说两句……争个半天，然后就锁了

**Comment by ghost** [comment]
数据太长，叠加的时候出现了OSError: [WinError 6] 句柄无效。一直在循环叠加

**Reply by allMagicNB** [reply]
好好好，九千七百多个字符
目前还是猜测，而且从日志看不出……

**Comment by ghost** [comment]
转码肯定有，但16进制代码不难转，json叠加再转以vb的速度也会很快，不至于卡那么久

**Comment by SuoHouDaiShi** [comment]
![图片](https://github.com/Hex-Dragon/PCL2/assets/119282947/daf04d61-8f4e-4b09-be6e-c3e27cfff59c)
AI 是这么说的……
<del>用微软的 AI 回答微软自家语言的问题，合理吧</del>
---
Edit:
![图片](https://github.com/Hex-Dragon/PCL2/assets/119282947/cbe50647-5980-4dce-bf30-203020cd6c8c)
